openapi/api.github.com/operations/actions/create-or-update-org-secret.json

```diff
@@ -0,0 +1,92 @@
+{
+  "summary": "Create or update an organization secret",
+  "description": "Creates or updates an organization secret with an encrypted value. Encrypt your secret using [LibSodium](https://libsodium.gitbook.io/doc/bindings_for_other_languages). You must authenticate using an access token with the `admin:org` scope to use this endpoint. GitHub Apps must have the `secrets` organization permission to use this endpoint.\n\nEncrypt your secret using the [tweetsodium](https://github.com/mastahyeti/tweetsodium) library.\n\n\n\nEncrypt your secret using [pynacl](https://pynacl.readthedocs.io/en/stable/public/#nacl-public-sealedbox) with Python 3.\n\n\n\nEncrypt your secret using the [Sodium.Core](https://www.nuget.org/packages/Sodium.Core/) package.\n\n\n\nEncrypt your secret using the [rbnacl](https://github.com/RubyCrypto/rbnacl) gem.",
+  "operationId": "actions/create-or-update-org-secret",
+  "tags": ["actions"],
+  "externalDocs": {
+    "description": "API method documentation",
+    "url": "https://developer.github.com/v3/actions/secrets/#create-or-update-an-organization-secret"
+  },
+  "parameters": [
+    {
+      "name": "accept",
+      "description": "Setting to `application/vnd.github.v3+json` is recommended",
+      "in": "header",
+      "schema": {
+        "type": "string",
+        "default": "application/vnd.github.v3+json"
+      }
+    },
+    {
+      "name": "org",
+      "description": "org parameter",
+      "in": "path",
+      "required": true,
+      "schema": { "type": "string" }
+    },
+    {
+      "name": "secret_name",
+      "description": "secret_name parameter",
+      "in": "path",
+      "required": true,
+      "schema": { "type": "string" }
+    }
+  ],
+  "responses": { "418": { "description": "Response definition missing" } },
```

Comment:
That is not correct, there are two responses:
https://developer.github.com/v3/actions/secrets/#response-when-creating-a-secret-1